src/main/resources/db/migration/changes/007-create-Instructors.json

```
@@ -2,7 +2,7 @@
     {
       "changeSet": {
         "id": "007-create-Instructors",
-        "author": "Hannya",
+        "author": "hannyaYan",
```

```suggestion
        "author": "Hannya",
```

Once a database migration file is merged into main, it cannot be changed again except through another database migration; otherwise, the database will become corrupted.